src/twisted/names/dns.py

```diff
@@ -1902,6 +1917,79 @@ def __hash__(self):
 
 
 
+@implementer(IEncodable, IRecord)
+class Record_SSHFP(tputil.FancyEqMixin, tputil.FancyStrMixin):
+    """
+    A record containing the fingerprint of an SSH key.
+
+    @type keytype: L{int}
+    @ivar keytype: The SSH key algorithm or key type.
+    Note that the numbering used for SSH key algorithms is specific
+    to the SSHFP record type, and is not the same as the nubering
+    used for KEY or SIG records.
+
+    @type fptype: L{int}
```

Comment:
I prefer to not use abbreviations and name it instead `fingerprintType` or something like that.